webapp/src/ts/modules/contacts/contacts-edit.component.html

```diff
@@ -10,23 +10,20 @@
   <div class="col-sm-8 item-content material" [hidden]="loadingContent || contentError">
     <div class="card">
       <mm-enketo formId="contact-form" [editing]="enketoContact?.docId" [status]="enketoStatus" (onSubmit)="save()" (onCancel)="navigationCancel()">
-        <div duplicate-contacts id="duplicate_contacts" [ngStyle]="{ display: duplicates.length > 0 ? 'block' : 'none' }" class="question invalid-value">
-          <p class="results_header invalid-value-msg active">{{ duplicateMessage }}</p>
-          <div class="divider"></div>
+        <div duplicate-contacts id="duplicate_contacts" *ngIf="duplicates.length" [ngClass]="{'invalid-value': enketoError}" class="question">
+          <p [ngClass]="{'invalid-value-msg': enketoError}" class="results_header active">{{ duplicateMessage }}</p>
```

Comment:
Varying the color here based on the presence of `enketoError`.  The idea is that when the user tries to submit the form and dupes are found, the dupes should be highlighted in red.  However, if the user then changes something in the form (e.g. updates to a different name value), then the `enketoError` is automatically cleared).  In that case, we can follow the standard form behavior and clear the red error highlighting (though we still leave the duplicate list for reference).  

My hope is that this gives a bit of a visual queue to the user that they can try submitting again and might get different results.